src/main/java/com/booleanuk/core/Exercise.java

```diff
@@ -16,26 +16,27 @@ public class Exercise {
 
     // 2. There are currently some upper case characters in the URL. Using an appropriate string method on the fixedUrl member above,
     // set the value of lowerCasedUrl.
-    public String lowerCasedUrl = "";
+    public String lowerCasedUrl = fixedUrl.toLowerCase();
 
 
     // 3. There is still white space on both ends of the URL! Use the appropriate string method to trim that white space
     // and set the value of the url member below
-    public String url = "";
+    public String url = lowerCasedUrl.strip();
 
 
     // 4. Using the appropriate string method on url, set the value of the protocol member below
-    public String protocol = "";
+    public String protocol = url.substring(0,5);
 
 
     // 5. Using the appropriate string method on url, set the value of the domain member below
-    public String domain = "";
+    public String domain = url.substring(8,21);
 
 
     // 6. Set the length member below to the length of the url member
-    public int length = 0;
+    public int length = url.length();
 
 
     // 7. Using concatenation and existing members, set the faqUrl member below to the faq page of the boolean website
-    public String faqUrl = "";
+    public String faqUrl = url.replace("who-we-are", "faq");
```

Comment:
Although this works, the instructions specify the use of existing members and concatenation.